lib/views/github-tile-view.js

```diff
@@ -2,32 +2,46 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Octicon from '../atom/octicon';
 
+import RefHolder from '../models/ref-holder';
+import Tooltip from '../atom/tooltip';
 import {addEvent} from '../reporter-proxy';
-import {autobind} from '../helpers';
+
+export const tooltipMessage = 'Click to view open GitHub pull requests';
```

Comment:
Should we remove the "Click to"? It feels a bit redundant and would make it quicker to read. Like on .com, it says what you can do (if you click):

![image](https://user-images.githubusercontent.com/378023/50205124-785a0580-03aa-11e9-8c50-eb5c8904d42e.png)

Maybe we can also mention the checkout feature which is a good reason to view a PR in Atom. Maybe:

`View and checkout GitHub pull requests`

Reply:
@simurai the push pull tooltip says "click to push / pull / whatever" so I was keeping the copy consistent with that.

That said, I'm not strongly attached to "click to".

Reply:
Yeah, I guess the "Click to fetch/pull/push" is somewhat needed to differentiate from the "Right-click for more"? Hmmm.. 🤔 Maybe it would be a good chance to teach what fetch/pull/push does? Like

- Fetch -> [Fetch to see if there are new commits | Right-click for more options]
- Pull -> [Pull to add new commits to your local branch | Right-click for more options]
- Push -> [Push to share your local commits | Right-click for more options]
- Publish -> [Publish to share your local branch | Right-click for more options]

This is probably technically wrong, but it's kinda how I would explain it. :joy: Below the other tiles:

- Branch -> [Create or switch branches]
- GitHub -> [View and checkout GitHub pull requests]
- Git -> [Stage, view and commit changes]

Feel free to merge with "Click to ...", the suggestions above probably need a bit more thought and testing with users.